flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/wrappers/StormTuple.java

```diff
@@ -44,16 +45,30 @@
 	/** The schema (ie, ordered field names) of the tuple */
 	private final Fields schema;
 
+	private final int taskId;
+	private final String producerStreamId;
+	private final MessageId id;
+	private final String producerComponentId;
+
+
+	/**
+	 * Constructor which sets defaults for producerComponentId, taskId, and componentID
+	 * @param flinkTuple the Flink tuple
+	 * @param schema The schema of the storm fields
+	 */
+	StormTuple(final IN flinkTuple, final Fields schema) {
+		this(flinkTuple, schema, -1, "testStream", "componentID");
+	}
```

Comment:
Ok. I guess it would make sense to use Storm's `Utils.DEFAULT_STREAM_ID` here? And maybe add a `public final static String DEFAULT_OPERATOR_ID` variable to `StormTuple`? What about using "defaultID" or "unspecified" instead of "componentID" or similar? Just to make it clear if the name shows up in the UI?